docs/3.0.0-beta.x/migration-guide/migration-guide-beta.17-to-beta.18.md

```diff
@@ -116,7 +116,7 @@ Then run either `yarn install` or `npm install`.
 
 ## Database configuration
 
-Now that you have installed the new database package. You need to update your `database.json` configuration files located in `./config/environments/{env}/database.json`.
+Now that you have installed the new database package, You need to update your `database.json` configuration files located in `./config/environments/{env}/database.json`.
```

Comment:
should be miniscule Y after a comma :)

Reply:
[Fixed!](https://github.com/strapi/strapi/pull/4895/commits/896b018f63c28ab874dd5afed98e674268773ce4)